cd/publish-chart.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eu
+
+# Decrypt a private SSH key having its public key registered on GitHub. It will
+# be used to establish an identity with rights to push to the repo hosting our
+# Helm charts: https://github.com/jupyterhub/helm-chart
+openssl aes-256-cbc -K $encrypted_c6b45058ffe8_key -iv $encrypted_c6b45058ffe8_iv -in cd/id_rsa.enc -out cd/id_rsa -d
+chmod 0400 cd/id_rsa
+
+docker login -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}"
+
+# Activate logging of bash commands now that the sensitive stuff is done
+set -x
+
+# As chartpress utilizes git to push to our Helm chart repository, we configure
```

Comment:
Minor English tip: approximately every time you might use "utilize", "use" is probably a better choice.